server.js

```diff
@@ -1,6 +1,9 @@
-import app from './src/app';
+import application from './src/app';
 const port = 3000;
 
-app.listen(port, () => {
-  console.log(`app running on port ${port}`);
-});
+application()
```

Comment:
I liked the ideia, I just updated based on your feedback https://github.com/waldemarnt/building-testable-apis-with-nodejs-code/pull/6/commits/fedf2e387c12e41ff2bb0d80d9065357b07e89a3